README.md

```diff
@@ -1 +1 @@
-![matthewctechnology](https://github.com/matthewctechnology/matthewctechnology/blob/main/images/banner.png)
+![matthewctechnology](/images/banner.png)
```

Comment:
this makes the repo more flexible as the repo is cloned, etc.. this will reference the repository